src/scss/main/_vantages.scss

```diff
@@ -0,0 +1,90 @@
+.vantages {
+  padding-top: 60px;
+  padding-bottom: 60px;
+  @include media-breakpoint-up(lg) {
+    padding-top: 100px;
+    padding-bottom: 100px;
+  }
+}
+.title-holder {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 50px;
+}
+.vantages__list-holder {
+  position: relative;
+}
+.vantages-button-holder {
+  display: flex;
+  position: absolute;
+  z-index: 2;
+  align-items: center;
+  height: 100%;
+  @include media-breakpoint-up(lg) {
+    padding-right: 15px;
+    padding-left: 15px;
+  }
+}
+.vantages-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  background-color: #fff0;
```

Comment:
add background: none for all buttons.

note: you can use `transparent` key word when you need to add transparent background.